infra/build_specified_commit.py

```diff
@@ -29,7 +29,25 @@
     'BuildData', ['project_name', 'engine', 'sanitizer', 'architecture'])
 
 
-def build_fuzzers_from_commit(commit, build_repo_manager, build_data):
+def copy_src_from_docker(project_name, host_dir):
+  """Copy /src from docker to the host."""
+  # Copy /src to host.
+  image_name = 'gcr.io/oss-fuzz/' + project_name
+  docker_args = [
+      '-v',
+      host_dir + ':/out',
+      image_name,
+      'cp',
+      '-r',
+      '/src',
+      '/out',
+  ]
+  helper.docker_run(docker_args)
+  return os.path.join(host_dir, 'src')
```

Comment:
I've forgotten what's inside the `host_dir`, so asking just in case, would it be safer to use the subdirectory name other than `src` for this?

Reply:
host_dir is caller supplied (and currently temp dir), so should be safe.